packages/api/es/textFields.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const textFields = [
+  'type',
+  'provider',
+  'granuleId',
+  'collectionId',
+  'pdrName',
+  'file',
+  'executions',
+  'version',
+  'name',
+  'id',
+  'status',
+  'operationType',
+  'taskArn',
+  'execution',
+  'address',
+  'originalUrl',
+  'cmrLink',
+  'protocol',
+  'host',
+  'location',
+  'workflow',
+  'state',
+  'arn',
+];
+
+/**
+ * For certain Elasticsearch aggregations, text fields are more efficient to use
+ * the keyword version, which is stored in [fieldname].raw. This function
+ * converts that field to use the keyword version if its a text field.
+ *
+ * @param {string/number} field - elasticsearch field
+ * @returns {string} - field converted to the right subfield if eligible
+ */
+function convertTextField(field) {
```

Comment:
Can you please write a unit test for this function?

Reply:
Done!